src/main/resources/db/changelog/db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet id="202010211812" author="MrRevo">
+        <createTable tableName="user">
+            <column name="id" type="bigint">
```

Comment:
albo to na UUID albo dorzuć pole UUID na id z keycloaka + indeks, dodatkowo tu możesz dać autoIncrement=true zamiast to definiować osobno nizej